src/main/java/teamroots/embers/tileentity/TileEntityEmberBore.java

```diff
@@ -164,10 +172,10 @@ public void update() {
             return;
         if (getWorld().isRemote)
             handleSound();
-        speedMod = UpgradeUtil.getTotalSpeedModifier(this, upgrades) * ConfigManager.emberBoreSpeedMod;
+        speedMod = UpgradeUtil.getTotalSpeedModifier(this, upgrades) * ConfigMachine.EMBER_BORE_CATEGORY.speedMod;
         lastAngle = angle;
         if (isRunning) {
-            angle += 12.0f * speedMod;
+            angle += (float) (12.0f * speedMod);
```

Comment:
* [ ] `angle += 12.0d * speedMod;`
       `angle` and `speedMod` are doubles so work with them